travvy/src/components/AttractionDetails.vue

```diff
@@ -3,24 +3,31 @@
     <!-- Button to go to go back to the AttractionsList page --> 
     <router-link to="/AttractionList" tag="button" class="back"><span>Back</span></router-link>
 
-    <h1>CN Tower </h1>
+    <h1>{{$store.state.recommendedAttractions[0].attraction_name}} </h1>
```

Comment:
I appreciate that you changed the code from static to dynamic - it will make the MVP much more useful!